creator-node/src/routes/nodeSync.js

```diff
@@ -204,6 +207,13 @@ module.exports = function (app) {
     return successResponse()
   }))
 
+  app.get('/sync_history', handleResponse(async (req, res) => {
+    const aggregateSyncData = await SyncHistoryAggregator.getAggregateSyncData()
```

Comment:
just observing that this route will throw 500 if either of the inner functions fail (which they can since they are not wrapped in try-catch) but this is probably fine since that is correct behavior anyway, just not ideal code

also a small comment/jsdoc for this would be helpful explaining its purpose and return schema

Reply:
ok, will wrap getters in try/catches, and add jsdoc comments with data structures

Reply:
perfect, thanks!